man/fi_getinfo.3.md

```diff
@@ -249,6 +250,72 @@ struct fi_info {
   closely associated with a hardware NIC.  See
   [`fi_nic`(3)](fi_nic.3.html) for details.
 
+*hmem_attr - heterogeneous memory attributes*
+: Optionally supplied HMEM attributes.  HMEM attributes may be
+  specified and returned as part of fi_getinfo.  When provided as
+  hints, requested values of struct fi_hmem_attr should be set.  On
+  output, the actual HMEM attributes that can be provided will be
+  returned.
+
+## HMEM ATTRIBUTES
+
+```c
+enum fi_hmem_attr_opt {
+	FI_HMEM_ATTR_ENABLED,
```

Comment:
I think naming it as FI_HMEM_ATTR_UNSPEC reflects what it means better. It's also better to make it start from 0 so it allows application to just create a `fi_mr_attr = {0}`